tests/func/test_run_single_stage.py

```diff
@@ -1032,3 +1032,18 @@ def test_run_force_doesnot_preserve_comments_and_meta(tmp_dir, dvc, run_copy):
 
     assert "comment" not in (tmp_dir / "bar.dvc").read_text()
     assert "meta" not in (tmp_dir / "bar.dvc").read_text()
+
+
+def test_run_noshell_skips_extra_quoting(dvc, mocker):
+    cmd = "Rscript -e 'saveRDS(1:10, \"ops-data.rds\")'"
```

Comment:
Since this is not running, let's try to keep it as simple as possible, so people are not encouraged to figure out what this does :slightly_smiling_face: e.g.
```suggestion
    cmd = "myscript arg1 arg2"
```

Reply:
Or there was some intention behind it?